File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/BrushViewChangeListener.java
package ja.burhanrashid52.photoeditor;

/**
 * Created on 1/17/2018.
 * @author <a href="https://github.com/burhanrashid52">Burhanuddin Rashid</a>
 * <p></p>
 */

interface BrushViewChangeListener {
    void onViewAdd(DrawingView drawingView);

    void onViewRemoved(DrawingView drawingView);

    void onStartDrawing();

    void onStopDrawing();
}

File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/BrushViewChangeListener.kt
package ja.mhdxbilal007.photoeditor

/**
 * Created on 1/17/2018.
 * @author [mhdxbilal007](https://github.com/mhdxbilal007)
 *
 *
 */
interface BrushViewChangeListener {
    fun onViewAdd(drawingView: DrawingView)
    fun onViewRemoved(drawingView: DrawingView)
    fun onStartDrawing()
    fun onStopDrawing()
}